pkgs/applications/networking/cluster/k3s/README.md

```diff
@@ -1,114 +1,26 @@
-# k3s versions
+# K3s
 
-K3s, Kubernetes, and other clustered software has the property of not being able to update atomically. Most software in nixpkgs, like for example bash, can be updated as part of a "nixos-rebuild switch" without having to worry about the old and the new bash interacting in some way.
+K3s is a simplified version of [Kubernetes](https://wiki.nixos.org/wiki/Kubernetes). It bundles all components for a kubernetes cluster into a few of small binaries.
 
-K3s/Kubernetes, on the other hand, is typically run across several NixOS machines, and each NixOS machine is updated independently. As such, different versions of the package and NixOS module must maintain compatibility with each other through temporary version skew during updates.
+## Usage
 
-The upstream Kubernetes project [documents this in their version-skew policy](https://kubernetes.io/releases/version-skew-policy/#supported-component-upgrade-order).
+Starter usage examples can be found in [docs/USAGE.md](docs/USAGE.md). 
```

Comment:
## [Usage examples](docs/USAGE.md)

Reply:
Fine with removing verbiage, but not a fan of inconsistent header usage. "Are they links or are they headers?" type thought. I am thinking just an array of links below each heading and at least be consistent with that in the README across all sections.

Reply:
Sure, can do like that. Keep it short.